lv2/atomic_compat.h

```diff
@@ -0,0 +1,64 @@
+/*
+  SPDX-License-Identifier: ISC
+
+  Sfizz LV2 plugin
+
+  Copyright 2019-2020, Paul Ferrand <paul@ferrand.cc>
```

Comment:
I'm not the author of this file 🙂 
On a side note I think there's no real need to track file copyright since we have a version control system, so we can just maintain the `LICENSE.md` file and author's list globally. I realize now that I updated the C++ library like this but kept my name in the lv2 plugin, I think this is unnecessary.

Reply:
It's not about the authorship, but copyright assignment.
When a project is assigned to one single person, it facilitates such things as update the license in the future, if there is ever need.

As I recall, GNU for instance requires copyrigth assignment from their contributors.

Reply:
Ok I see your point, but I'm not super comfortable asking for copyright assignments. Since we're discussing code style and contibution guidelines in general, I propose we move the discussion on this there since it's not really the heart of the PR.